src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -229,27 +231,32 @@ private List<String> scriptLauncherCmd(ShellController c, FilePath ws, @CheckFor
         if (os == OsType.WINDOWS) { // JENKINS-40255
             scriptPath = scriptPath.replace("\\", "/"); // cygwin sh understands mixed path  (ie : "c:/jenkins/workspace/script.sh" )
         }
+        String scriptPathCopy = scriptPath + ".copy"; // copy file to protect against "Text file busy", see JENKINS-70874
```

Comment:
Ought to be covered by https://github.com/jenkinsci/durable-task-plugin/blob/8ea90b6f96daddccc18e4622eb289675247e213c/src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java#L487 I think.